infrastructure/rdfdb/docker/scripts/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+echo 'org.apache.tomcat.util.digester.PROPERTY_SOURCE=org.apache.tomcat.util.digester.EnvironmentPropertySource' >> /usr/local/tomcat/conf/catalina.properties
+echo "RDF4J_DATA_DIR is set to: ${RDF4J_DATA_DIR}"
+
+/usr/local/tomcat/modify_web_xml.sh
```

Comment:
I am curious why tomcat was chosen.  Is it because that was used in the past?  I don't know if it is better then jetty or others - As long as it is performant I don't have an opinion one way or the other.

Reply:
Good question... The original commit had a .gitignore on the Dockerfile, so that is why it wasn't apparent.   I added the Dockerfile as well to this PR.   The Dockerfile using a standard image for RDF4J which runs the database in Tomcat.   We could change it later, if need be to Jetty though and create a custom image.

Reply:
I was asking the same question. I think Jetty is cleaner but having it built in makes this simpler.